kafka/consumer/simple.py

```diff
@@ -353,6 +355,13 @@ def _fetch(self):
                     # Retry this partition
                     retry_partitions[resp.partition] = partitions[resp.partition]
                     continue
+                except FailedPayloadsError:
+                    log.warning("Failed payloads of %s"
+                                "Resetting partition offset...",
+                                resp.failed_payloads)
+                    # Retry this partition
+                    retry_partitions[resp.failed_payloads.partition] = partitions[resp.failed_payloads.partition]
```

Comment:
.failed_payloads has been changed to .payload .  suggest using `e.payload.partition`